### PR TITLE
Add list repo org variables and secrets

### PR DESCRIPTION
**Pull request by tomfeigin**
This wraps two repository API endpoints which are currently not available in the interface:

- [List repository organization actions secrets ](https://docs.github.com/en/enterprise-cloud@latest/rest/actions/secrets?apiVersion=2022-11-28#list-repository-organization-secrets)
- [List repository organization actions variables](https://docs.github.com/en/rest/actions/variables?apiVersion=2022-11-28#list-repository-organization-variables)

### CODE REVIEW

**Review by gmlewis — APPROVED** [review]
Thank you, @tomfeigin !
LGTM.

Awaiting second LGTM+Approval from any other contributor to this repo before merging.

**Review by fchimpan — APPROVED** [review]
Great job! Thanks @tomfeigin
LGTM🎉

**Comment by gmlewis** [comment]
Thank you, @fchimpan !
Merging.

**Pull request merged**
